tests/helpers.py

```diff
@@ -2372,6 +2372,14 @@ def storagecluster_independent_check():
     )
 
 
+def retrieve_rgw_storageclass_name():
```

Comment:
I recommend to move this method to `ocs_ci/ocs/resources/rgw.py` so that it is at one place with other rgw related things. Also this method is only helpful only for rgw tests so it shouldn't be in generic helpers module.